### PR TITLE
SorrTask447 Convert Telegram GDoc to MD

### PR DESCRIPTION
**Pull request by Dylan-Celli**
Task #447

### CODE REVIEW

**Comment by samarth9008** [comment]
Resolve the conflict. We have moved the `Readme` file from that location. Merge the latest master.

**Comment by Dylan-Celli** [comment]
> Resolve the conflict. We have moved the `Readme` file from that location to root of the repo. Merge the latest master.

Ok, I merged master back into this branch

File: docs/Telegram.md
Comment (DanilYachmenev) [inline docs/Telegram.md:1]: Add a reference to this Doc to the [README](https://github.com/sorrentum/sorrentum/blob/master/README.md) under `In-depth` docs section

File: docs/Telegram.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    Telegram cloud. This means you can only access messages in a secret chat
+    from their device of origin.
+
+## Regular
+
+- We use regular chats for
+  - General discussions within a team
+  - Group chats
+- We do not share sensitive information via regular chats
+<!-- Regular chats cannot be accessed via Telegram Desktop version -->
```
Comment (DanilYachmenev) [inline docs/Telegram.md:48]: remove this line, its not true and now is hidden at all

**Review by DanilYachmenev — COMMENTED** [review]
sg, see couple nits and ready

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

**Pull request merged**